pkg/didcomm/protocol/didexchange/persistence/connection.go

```diff
@@ -29,24 +35,66 @@ type ConnectionRecorder struct {
 //
 // Args:
 //
-// key: key for the invitation object to be saved in store
-// value: invitation object
+// verKey: recipient key of the invitation object to be saved in store
+// invitation: invitation to be stored
 //
 // Returns:
 //
 // error: error
-func (c *ConnectionRecorder) SaveInvitation(key string, value interface{}) error {
-	bytes, err := json.Marshal(value)
+func (c *ConnectionRecorder) SaveInvitation(verKey string, invitation *models.Invitation) error {
+	k, err := invitationKey(verKey)
 	if err != nil {
 		return err
 	}
 
-	storeKey, err := computeHash([]byte(key))
+	bytes, err := json.Marshal(invitation)
 	if err != nil {
 		return err
 	}
 
-	return c.store.Put(storeKey, bytes)
+	return c.store.Put(k, bytes)
+}
+
+// FindInvitation finds invitation for given key from underlying store and
+// stores the result in the value pointed to by v
+//
+// Args:
+//
+// verKey: recipient key of the invitation object to be saved in store
+//
+// Returns:
+//
+// invitation found
+// error: error
+func (c *ConnectionRecorder) FindInvitation(verKey string) (*models.Invitation, error) {
```

Comment:
`Find` isn't our usual prefix for getting or looking up records.

Reply:
I agree

Reply:
Will rename it to `Get`

Reply:
done